lib/bourbon.rb

```diff
@@ -1,20 +1,3 @@
 require "bourbon/generator"
-
-module Bourbon
-  if defined?(Rails) && defined?(Rails::Engine)
-    class Engine < ::Rails::Engine
-      require "bourbon/engine"
-    end
-
-    module Rails
-      class Railtie < ::Rails::Railtie
-        rake_tasks do
-          load "tasks/install.rake"
-        end
-      end
-    end
-  else
-    bourbon_path = File.expand_path("../../core", __FILE__)
-    ENV["SASS_PATH"] = [ENV["SASS_PATH"], bourbon_path].compact.join(File::PATH_SEPARATOR)
-  end
-end
+bourbon_path = File.expand_path("../../core", __FILE__)
```

Comment:
I think it cannot -- the relative path + __FILE__ wonkiness that's necessary to get an absolute path means that `./core` would probably resolve to `bourbonlib/core`, not `bourbon/core`.